tests/integration/suite/test_features.py

```diff
@@ -27,3 +27,9 @@ def test_can_list(admin_mc, user_mc, remove_resource):
     admin_client = admin_mc.client
     admin_client.list_feature()
     assert True
+
+
+def test_default_set_as_value(admin_mc):
+    """Asserts feature values reflect default prior to being toggled"""
+    istio_feature = admin_mc.client.by_id_feature("istio-virtual-service-ui")
+    assert istio_feature["value"] == istio_feature["status"]["default"]
```

Comment:
Update to validate that the value is not None, this is not guaranteed though as this feature being flipped elsewhere will cause a false positive. Potentially marking as non-parallel and removing the value to ensure only the default will get filled in.